ansible/group_vars/all

```diff
@@ -124,6 +127,13 @@ controller:
       seedNodes: "{{ groups['controllers'] | map('extract', hostvars, 'ansible_host') | list }}"
   # We recommend to enable HA for the controllers only, if bookkeeping data are shared too. (localBookkeeping: false)
   ha: "{{ controller_enable_ha | default(True) and groups['controllers'] | length > 1 }}"
+  protocol: "{{ controller_protocol | default('https') }}"
```

Comment:
Can we make the default http instead of https? Helps for local development environment 

You can pass override to ansible-playbook with -e flag in Travis if you want to enable in Travis testing deploy script

Reply:
Agree. Flipping this default breaks our documented workaround for wsk -i which instructs a user to set the api host against the controller directly.